rest/src/test/scala/tamer/rest/Fixtures.scala

```diff
@@ -0,0 +1,67 @@
+package tamer.rest
+
+import io.circe.generic.semiauto.deriveCodec
+import tamer.{AvroCodec, HashableState}
+import zio.interop.catz._
+import zio.interop.catz.implicits._
+import zio.test._
+import zio.{Task, ZIO, _}
+
+import java.util.UUID
+import scala.util.Random
+
+object Fixtures {
+  import org.http4s._
+  import org.http4s.dsl.Http4sDsl
+  import org.http4s.implicits._
+  import org.http4s.server.Router
+  import org.http4s.server.blaze._
+
+  case class Key(time: Long)
+  object Key {
+    implicit val codec = AvroCodec.codec[Key]
+  }
+  case class Value(time: Long, uuid: UUID)
+  object Value {
+    implicit val codec      = AvroCodec.codec[Value]
+    implicit val circeCodec = deriveCodec[Value]
+  }
+  case class State(i: Int)
+  object State {
+    implicit val codec = AvroCodec.codec[State]
+    implicit val hash = new HashableState[State] {
+      override def stateHash(s: State): Int = s.i
```

Comment:
Isn't this supposed to be used as (part of) the partition key used for the state in kafka? If so we need it stable, it should not change depending on the content of the state